docs/excel/excel-add-ins-workbooks.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Work with workbooks using the Excel JavaScript API
+description: ''
+ms.date: 11/26/2018
+---
+
+
+# Work with Workbooks using the Excel JavaScript API
+
+This article provides code samples that show how to perform common tasks with workbooks using the Excel JavaScript API. For the complete list of properties and methods that the **Workbook** object supports, see [Workbook Object (JavaScript API for Excel)](https://docs.microsoft.com/javascript/api/excel/excel.workbook). This article also covers workbook-level actions performed through other similarly scoped objects, such as [Application](https://docs.microsoft.com/javascript/api/excel/excel.application).
+
+The workbook object is the entry point for your add-in to interact with Excel. It contains a [WorksheetCollection](https://docs.microsoft.com/javascript/api/excel/excel.worksheetcollection) to manipulate individual worksheets. Most work done to [ranges](excel-add-ins-ranges.md) happens directly through a worksheet. The article [Work with Worksheets using the Excel JavaScript API](excel-add-ins-worksheets.md) describes how to access and edit worksheets.
```

Comment:
I find "manipulate" imprecise (and arguably not correct). The collection can be used to add worksheets, navigate among them, and assign handlers to worksheet events.